runtime/src/main/java/org/corfudb/security/tls/SslContextConstructor.java

```diff
@@ -33,11 +33,11 @@ public static SslContext constructSslContext(boolean isServer,
                                                  String ksPasswordFile,
                                                  @NonNull String trustStorePath,
                                                  String tsPasswordFile) throws SSLException {
-        log.info("Construct ssl context based on the following information:");
-        log.info("Key store file path: {}.", keyStorePath);
-        log.info("Key store password file path: {}.", ksPasswordFile);
-        log.info("Trust store file path: {}.", trustStorePath);
-        log.info("Trust store password file path: {}.", tsPasswordFile);
+        log.trace("Construct ssl context based on the following information:");
```

Comment:
The presence of this message helped distinguish a firewall misconfiguration from a connectivity issue.
What had happened is that the Corfu Server port was blocked but SSL which uses 80/443 went through fine. So wondering if this message can still remain?

Reply:
Sundar, these messages in the log are literally a flood. Even if it helped to find a bug once I don't believe that the price we are paying for that is worth it.
These messages we print so often that every single time when you read logs you read these messages and it gets pretty hard to find all other messages in the log.

Reply:
Slava, your arguments are very valid and I like the thought process. Please have a work around for these issues as we just cannot afford to lose out on debug ability of the product.